src/world.rs

```diff
@@ -413,10 +413,25 @@ impl World {
     }
 
     #[inline(always)]
-    pub(crate) fn entities_meta(&self) -> &[EntityMeta] {
+    pub(crate) fn entities_meta(&self) -> &Vec<EntityMeta> {
         &self.entities.meta
     }
 
+    /// Get the id's of all pending (free) entities
+    pub fn pending(&self) -> &Vec<u32> {
+        &self.entities.pending
+    }
+
+    /// Set a custom list of generations
+    pub fn push_generations(&mut self, generations: &[u32]) {
+        self.entities.push_generations(generations);
+    }
+
+    /// Manually set what entities are pending
+    pub fn push_pending(&mut self, pendings: &[u32]) -> Result<(), PendingPushError>{
+        self.entities.push_pending(pendings)
+    }
```

Comment:
Is there a use case for calling these on a pre-existing world, or multiple times, or calling one but not the other? If not, these should be arguments to an alternative constructor, not methods.

Reply:
Moved both of them into one method.